ldes-fragmentisers/ldes-fragmentisers-geospatial/README.MD

```diff
@@ -0,0 +1,91 @@
+# Geospatial fragmentation
+
+Geospatial fragmentation will create fragments based on geospatial tiles selected of the `fragmenterProperty`.
+This allows you to fragment the data on geolocations.
+
+## Properties
+
+  ```yaml
+  name: "geospatial"
+  config:
+    maxZoomLevel: { Mandatory: Required zoom level }
+    fragmenterProperty: { Mandatory: defines which property will be used for bucketizing }
+    fragmenterSubjectFilter: { Optional: regex to filter the subjects matching the fragmenterProperty }
+  ```
+
+## Algorithm
+
+1. The fragmentationObjects of the member are determined
+   - We filter the statements where the predicate matches the `fragmenterProperty`
+   - If an optional regex is provided through the `fragmenterSubjectFilter` property, we filter on subjects that match this regex.
+   - We select all the object that pass the above filters.
+2. A bucket of tiles is created using the coordinates and provided zoomLevel. [This is a well known algorithm](https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#Lon..2Flat._to_tile_numbers_2)  
+3. The tiles are iterated. The member is added to every tile. Taking into account:
```

Comment:
Members are not always added to a tile fragment. Only if geospatial fragmentation is the lowest level of the fragmentation strategy.